testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/oauth/OIDCProtocolMappersTest.java

```diff
@@ -154,6 +159,8 @@ public void testTokenMapping() throws Exception {
             IDToken idToken = oauth.verifyIDToken(response.getIdToken());
             assertNotNull(idToken.getAddress());
             assertEquals(idToken.getName(), "Tom Brady");
+            assertNotNull(idToken.getOtherClaims().get("empty"));
+            assertNotNull(idToken.getOtherClaims().get("null"));
```

Comment:
Sorry, still not finished yet. It is retuning a string containing the word "null". I've changed it to hamcrest matcher isEmpty() and discovered that - working to fix it .